site/content/3.13/data-science/graphrag/services/importer.md

```diff
@@ -26,100 +26,43 @@ creates a structured knowledge graph. This graph is then imported into your
-    - `title`: The main theme or focus of the community.
-    - `summary`: A concise overview of the community's central narrative.
-    - `rating`: A numerical score indicating the community's significance (the higher, the better).
-    - `rating_explanation`: Justification for the rating.
-    - `findings`: An array of detailed analyses, each containing a summary and explanation of key aspects.
-  - `level`: The hierarchical level of the community (e.g., `1` for top-level communities).
-  - `occurrence`: A normalized score (ranging from `0` to `1`) showing the relative frequency with which this community is mentioned or identified throughout your documents. A value close to 1 means this community is very common in your data and a value near `0` means it is rare.
-  - `sub_communities`: References to more specific sub-communities that are part of this larger community.
-- **Usage**: Enables you to:
-  - Identify and analyze major narrative threads and themes.
-  - Understand complex relationships between characters and concepts.
-  - Track the significance and impact of different story elements.
-  - Navigate through hierarchical relationships between themes.
-  - Discover patterns and recurring elements in your documents.
-
-### Relations Collection
-
-- **Purpose**: Stores relationships between different nodes in the graph.
-- **Key Fields**:
-  - `_from`: Source node reference.
-  - `_to`: Target node reference.
-  - `type`: Type of relationship (e.g., **PART_OF**, **MENTIONED_IN**, **RELATED_TO**, **IN_COMMUNITY**).
-  - Additional metadata depending on relationship type (Entity to Entity):
-    - `weight`: Relationship strength (the higher, the better).
-    - `description`: Description of the relationship.
-    - `source_id`: Source of the relationship.
-    - `order`: Order of the relationship.
-- **Usage**: Enables traversal and analysis of relationships between different elements.
+{{< tip >}}
+You can also use the GraphRAG Importer service via the ArangoDB [web interface](../web-interface.md).
+{{< /tip >}}
 
-### Relationship Types
+## Creating a new project
+
+To create a new GraphRAG project, use the `CreateProject` method by sending a
+`POST` request to the `genai/v1/project` endpoint. You must provide a unique
+`project_name` and a `project_type` in the request body. Optionally, you can
+provide a `project_description`.
+
+```curl
+curl -X POST "https://<ExternalEndpoint>:8529/gen-ai/v1/project" \
+-H "Content-Type: application/json" \
+-d '{
+  "project_name": "docs",
+  "project_type": "graphrag",
+  "project_description": "A documentation project for GraphRAG."
+}'
+```
+All the relevant ArangoDB collections (such as documents, chunks, entities,
+relationships, and communities) created during the import process will
+have the project name as a prefix. For example, the Documents collection will
+become `<project_name>_Documents`. The Knowledge Graph will also use the project
+name as a prefix.
```

Comment:
@mdawas @maxkernbach What happens if no project is given? (I know that the default project will be selected). So to be more precise: What affects does the default project have on prefixes of collections? No prefix at all?

Reply:
Thanks Max. Ok if we add this small addition to the docs we're good with this PR.